utility/utils.py

```diff
@@ -382,13 +382,7 @@ def configure_logger(test_name, run_dir, level=logging.INFO):
     )
     _handler.setFormatter(formatter)
     _root.addHandler(_handler)
-
-    url_base = "http://magna002.ceph.redhat.com/cephci-jenkins"
-    run_dir_name = run_dir.split("/")[-1]
-    log_url = "{url_base}/{run_dir}/{log_name}".format(
-        url_base=url_base, run_dir=run_dir_name, log_name=full_log_name
-    )
-
+    log_url = full_log_name
```

Comment:
you have removed `url_base`, any reason wjy ?

Reply:
Hi Rakesh,
url_base was used for formulating the absolute path of the log_file.
As I was giving a relative path for log files, Url_base was becoming an unused variable so I have removed it.
I have removed the whole block of code which is appending the url_base,dir_name and full_log_name
As we click on the log file link from resuts.html page (URL: http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-1614674135116/result.html) with only log_file_name in href. it gets appended to http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-1614674135116/ + <log_file_name> .
Log_url gets formulated from the **current page url/<logfile_name>**.

Please find the sample test executed after changing this code:
http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-1614925690996/